### ISSUE TITLE
Adding not existing user into a group throws NPE

### ISSUE THREAD

**Issue by NShaforostov** [epam/NGB#246]
Seems to be introduced by #219

Example command:

```
ngb reg_group asdf -u sdsd@ex.com
Exception in thread "main" java.lang.NullPointerException
        at com.epam.ngb.cli.manager.command.handler.http.AbstractHTTPCommandHandler.loadListOfUsers(AbstractHTTPCommandHandler.java:438)
        at com.epam.ngb.cli.manager.command.handler.http.CreateUserGroupHandler.parseAndVerifyArguments(CreateUserGroupHandler.java:86)
        at com.epam.ngb.cli.manager.command.CommandManager.run(CommandManager.java:90)
        at com.epam.ngb.cli.app.Application.createAndLaunchCommandManager(Application.java:255)
        at com.epam.ngb.cli.app.Application.run(Application.java:217)
        at com.epam.ngb.cli.app.Application.main(Application.java:203)
```

**Comment by SilinPavel** [comment]
@vpolyakov88  @NShaforostov  fixed in ddf7018.

**Comment by vpolyakov88** [comment]
tested
